mehalsgmues/templates/mag/stats/subscription_counter.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+{% load juntagrico.config %}
+{% block content %}
+    <div style="margin-top: 20px; display: flex; flex-direction: column; align-items: center;">
+        <div style="position: relative">
+            <img src="/static/img/EAT_counter_background.png" style="width: 300px;" />
+            <img src="/static/img/EAT_zeiger.png" style="width: 135px; position: absolute; left: 53px; bottom: -10px; transform-origin: calc(100% - 35px) 50%; transform: rotate({{ rotation }}deg);" />
+        </div>
+        <div style="margin-top: 26px; font-size: 34px; font-weight: bold;">{{ num_eat_equivalent|floatformat:1 }} / {{ target_num_eat }}</div>
+        <div style="margin-top: 10px;">{% blocktrans %}Wir haben diese Saison <b>bislang umgerechnet {{ num_eat_equivalent }} "EAT Standard"</b> verkauft. Um unser Saison-Ziel von {{ target_num_eat }} zu erreichen <b>sollten wir noch {{ missing_eat }} EAT an die Leute bringen.</b> {% endblocktrans %}</div>
```

Comment:
you can handle the rounding here like this:
```suggestion
        <div style="margin-top: 10px;">{% blocktrans with n=num_eat_equivalent|floatformat:1 missing=missing_eat|floatformat:1 trimmed %}
            Wir haben diese Saison <b>bislang umgerechnet {{ n }} "EAT Standard"</b> verkauft.
            Um unser Saison-Ziel von {{ target_num_eat }} zu erreichen <b>sollten wir noch {{ missing }} EAT an die Leute bringen.</b>
        {% endblocktrans %}</div>
```

Reply:
https://github.com/mehalsgmues/juntagrico-mag/pull/12/commits/13110ce5e54bd075a5626fd1223a9cc0cf7035dd